ztag/schema.py

```diff
@@ -510,15 +510,47 @@ class CensysString(WhitespaceAnalyzedString):
 })
 
 # The oracle ztag transform is a plain copy of the "handshake" field.
+# TODO: Apply some transform to replace all String() with CensysString()?
 ztag_oracle = zgrab2_oracle.oracle_scan_response["result"]["handshake"]
 
+
+# TODO: Apply some transform to replace all String() with CensysString()?
 ztag_mssql = SubRecord({
     "version": String(),
     "instance_name": String(),
     "encrypt_mode": Enum(values=zgrab2_mssql.ENCRYPT_MODES),
     "tls": zcrypto.TLSHandshake(doc="The TLS handshake with the server (for non-encrypted connections, this used only for the authentication phase).")
 })
 
+
+# TODO: Apply some transform to replace all String() with CensysString()?
+ztag_mysql = SubRecord({
+    "protocol_version": zgrab2.mysql.mysql_scan_response["result"]["protocol_version"],
+    "server_version": zgrab2.mysql.mysql_scan_response["result"]["server_version"],
+    "capability_flags": zgrab2.mysql.mysql_capability_flags,
+    "status_flags": zgrab2.mysql.mysql_server_status_flags,
+    "error_code": zgrab2.mysql.mysql_scan_response["result"]["error_code"],
+    "error_message": zgrab2.mysql.mysql_scan_response["result"]["error_message"],
+    "tls": zcrypto.TLSHandshake(doc="TODO"),
+})
+
+
+# TODO: Apply some transform to replace all String() with CensysString()?
+ztag_postgres = SubRecord({
+    "supported_versions": String(),  # TODO: CensysString?
```

Comment:
Can you provide an example of what goes into this? I'm a bit surprised to see it as a `String()` given version**s**.

Reply:
@justinbastress ?

Reply:
This comes from an error when connecting with a bogus client version of 0.0, e.g. `"FATAL:  unsupported frontend protocol 0.0: server supports 1.0 to 3.0"`.